### ISSUE TITLE
loss turns into nan when training using gpu

### ISSUE THREAD

**Issue by senochow** [alexander-rakhlin/CNN-for-Sentence-Classification-in-Keras#7]
When i ran this code in my machine,  the loss turns into nan ? how can i solve it?
Epoch 1/10
9595/9595 [==============================] - 1s - loss: 0.6242 - acc: 0.6295 - val_loss: 0.5265 - val_acc: 0.7470
Epoch 2/10
9595/9595 [==============================] - 1s - loss: nan - acc: 0.4117 - val_loss: nan - val_acc: 0.0000e+00
Epoch 3/10
9595/9595 [==============================] - 1s - loss: nan - acc: 0.0000e+00 - val_loss: nan - val_acc: 0.0000e+00
Epoch 4/10
9595/9595 [==============================] - 1s - loss: nan - acc: 0.0000e+00 - val_loss: nan - val_acc: 0.0000e+00
Epoch 5/10
9595/9595 [==============================] - 1s - loss: nan - acc: 0.0000e+00 - val_loss: nan - val_acc: 0.0000e+00
Epoch 6/10
9595/9595 [==============================] - 1s - loss: nan - acc: 0.0000e+00 - val_loss: nan - val_acc: 0.0000e+00
Epoch 7/10
9595/9595 [==============================] - 1s - loss: nan - acc: 0.0000e+00 - val_loss: nan - val_acc: 0.0000e+00

**Comment by alexander-rakhlin** [comment]
Maybe learning rate is too big. If you haven't change anything, the only explanation I see is your environment is somewhat different. Maybe you're using Tensorflow while my code was tested for Theano, etc.
